test/cpp/naming/resolver_component_test.cc

```diff
@@ -175,7 +202,7 @@ gpr_timespec n_sec_deadline(int seconds) {
                       gpr_time_from_seconds(seconds, GPR_TIMESPAN));
 }
 
-void poll_pollset_until_request_done(args_struct *args) {
+void PollPollsetUntilRequestDone(args_struct *args) {
```

Comment:
There are still more C-style function names below that need to be converted to C++-style names.